tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,240 @@
+		Expect(err).ToNot(HaveOccurred())
+		Expect(kubeController.CreateOrUpdateSigningSecret(
+			publicKey, publicSecretNameAuth, managedNamespace)).To(Succeed())
+
+		defaultEcPolicy = ecp.EnterpriseContractPolicySpec{
+			Description: "Red Hat's enterprise requirements",
+			PublicKey:   string(publicKey),
+			Sources: []ecp.Source{
+				{
+					Name: "ec-policies",
+					Policy: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//policy",
+					},
+					Data: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//data",
+					},
+				},
+			},
+			Exceptions: &ecp.EnterpriseContractPolicyExceptions{
+				NonBlocking: []string{"tasks", "attestation_task_bundle", "java", "test", "not_useful"},
+			},
+		}
+
+	})
+
+	AfterAll(func() {
+		// Delete the dev and managed namespaces with all the resources created in them
+		Expect(framework.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+		Expect(framework.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+	})
+
+	var _ = Describe("Creation of the 'Happy path for defult pipeline bundle' resources", func() {
+
+		It("Create a config map with the given detail in dev namespace.", func() {
+			_, err := framework.CommonController.CreateConfigMap(cm, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create ReleasePlan in dev namespace.", func() {
+			_, err := framework.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates Release Strategy with a given name in managed namespace.", func() {
+			_, err := framework.ReleaseController.CreateReleaseStrategy(releaseStrategyDefaultName, managedNamespace, releasePipelineNameDefault, releasePipelineBundleDefault, releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategyDefault)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create ReleasePlanAdmission in managed namespace.", func() {
+			_, err := framework.ReleaseController.CreateReleasePlanAdmission(targetReleasePlanAdmissionName, devNamespace, applicationNameDefault, managedNamespace, "", "", releaseStrategyDefaultName) //releaseEnvironment
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates EnterpriseContractPolicy in managed namespace.", func(ctx SpecContext) {
+			_, err := framework.TektonController.CreateEnterpriseContractPolicy(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy)
+			Expect(err).NotTo(HaveOccurred())
+		}, SpecTimeout(EnterpriseContractPolicyTimeout))
+
+		It("creates pvc with a given name in managed namespace with the given accessmode.", func() {
+			_, err := framework.TektonController.CreatePVCInAccessMode(releasePvcName, managedNamespace, corev1.ReadWriteOnce)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates sevice account with a given name in the manged namespace.", func() {
```

Comment:
Not going to keep posting this comment, but I think you should remove `with a given name` from all of these It lines

Reply:
I've removed this magical sentience
what was I thinking of ?